### PR TITLE
Done

### PR DESCRIPTION
**Pull request by madifana**

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -3788,6 +3788,14 @@ a:visited {
   background-image: url("http://i.imgur.com/oHSUiQq.jpg");
 }
 
+#kenneth-holmquist-cover {
+  background-image: url(http://i.imgur.com/28LWthk.jpg);
```
Comment (bhollan) [inline css/styles.css]: Could you put `"`s around your URLs?  It's important for consistency.

File: css/styles.css [hunk 2]
```diff
@@ -3788,6 +3788,14 @@ a:visited {
...
+}
+
+#kenneth-holmquist-card {
+  background-image: url(http://i.imgur.com/nNwDLtz.jpg);
```
Comment (bhollan) [inline css/styles.css]: Same thing here.

**Comment by bhollan** [comment]
Hi @madifana!

I think we're going to close this for now.  It's still here and ready to go in whenever you're ready to get the requested changes in.  Feel free to Ask A Question for help and/or re-open this PR whenever you're ready.

CC: @aturkewi

File: css/styles.css (hunk 2 above)
Comment (madifana) [inline css/styles.css]: Requested changes have been completed.

**Pull request closed without merging**